### ISSUE TITLE
Question about pre-training

### ISSUE THREAD

**Issue by zhandand** [SukwonYun/LTE4G#2]
hello, Your work is great, and I am very interested in it now. But I've met some problems about the pretraining. Could you please cite me the code about how the H^pre is utilized in expert training phrase, or could you tell me the relationships among encoder, classifier_og, expert_dict[sep] in Class modeler?

**Comment by SukwonYun** [comment]
Hi, @zhandand. Thank you for your interest in our work!

1. **Usage of H^pre**
   You can refer to the code below. As the pretraining has occurred using focal loss, we expect that the node embeddings, which are inputs for the expert models, will contain more balanced information.  
https://github.com/SukwonYun/LTE4G/blob/cd17498ee553ab120249fbee68d8bb9a3314a38b/models/lte4g.py#L431

2. **Relationships among encoder, classifier_og, expert_dict[sep]**
   - **encoder**: This encoder encodes features into a low-dimensional embedding space using focal loss. The node embeddings obtained from this encoder are used for training the experts.
   - **classifier_og**: Used for the initial training of the original graph. Recall that we use focal loss to train the encoder. However, as we tackle a classification task, we require a corresponding classifier to predict the node label. It's also used when selecting candidates for the class prototype to involve high-confidence nodes.
   - **expert_dict[sep]**: This represents the expert models responsible for 'HH', 'HT', 'TH', 'TT'. After training these experts, we distill knowledge to the class-wise 'H' and 'T' students, who are responsible for the final inference phase.

**Comment by zhandand** [comment]
ok, I got it now. Thank you so much for your detailed explanation.
